.github/workflows/monthly.yml

```diff
@@ -0,0 +1,110 @@
+name: Monthly Checks
+
+on:
+  schedule:
+    - cron:  '* * 1 * *'
+  workflow_dispatch:
+    inputs:
+      file:
+        description: 'File to test'
+        default: tests/README-test.md
+
+jobs:
+  create-issue:
+    if: always()
+    needs: [check_syntax, check_links, check_github_commit_dates]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v2
+        with:
+          name: result          
+      - name: Create Issue template
+        run: |
+          printf '%s\n%s%s %s\n%s\n%s\n' '---' 'title: Monthly Checks - ' $( date +"%B %Y" ) 'labels: automated issue' '---' > .github/ISSUE_TEMPLATE.md
+          printf '%s/%s%s%s' ${GITHUB_SERVER_URL} ${GITHUB_REPOSITORY} '/actions/runs/' ${GITHUB_RUN_ID} >> .github/ISSUE_TEMPLATE.md
+          echo -e \n***\n >> .github/ISSUE_TEMPLATE.md
+          cat github_commit_dates.md >> .github/ISSUE_TEMPLATE.md || True
+      - name: Verify template
+        run: cat .github/ISSUE_TEMPLATE.md
+      - name: create issue
+        id: create-iss
+        uses: buluma/create-an-issue@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: 'echo Created issue number ${{ steps.create-iss.outputs.number }}'
+
+  check_syntax:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+      - name: Setup Checks
+        run: |
+          cd tests 
+          npm install --silent chalk
+          cd ..
+      - name: Set env
+        run: echo "FILE=${{ github.event.inputs.file || 'README.md' }}" >> $GITHUB_ENV; 
```

Comment:
should always be `echo "FILE='README.md' >> $GITHUB_ENV; ` but I think this is redundant anyway? Just use `github.event.inputs.file` in the later steps?